src/integration/java/org/humancellatlas/ingest/submission/web/ProjectStatusUpdateTest.java

```diff
@@ -49,12 +60,29 @@ public void test_statusIsInProgress_afterSubmissionCreation() throws Exception {
         Project project = createProject();
         String submissionUrl = createSubmission();
         connectSubmissionToProject(project, submissionUrl);
-        verifyProjectStatus(project, WranglingState.IN_PROGRESS);
+        verifyProjectStatus(project, IN_PROGRESS);
+    }
+
+    @Test
+    public void test_statusIsSubmitted_afterSubmissionIsExported() throws Exception {
+        Project project = createProject();
+        String submissionUrl = createSubmission();
+        connectSubmissionToProject(project, submissionUrl);
+        setSubmissionToExported(submissionUrl);
+        verifyProjectStatus(project, SUBMITTED);
+    }
+
+
+
+    private void setSubmissionToExported(String submissionUrl) throws Exception {
+        webApp.perform(
+                put(submissionUrl + Links.COMMIT_EXPORTED_URL)
+        ).andExpect(status().isAccepted());
     }
 
     private void verifyProjectStatus(Project project, WranglingState wranglingState) {
```

Comment:
can you please change the signature of this to be:
```
private void assertProjectWrangingState(Project project, WranglingState expectedState)
```

Reply:
Do you mean `assert...` instead of `verify...`?